cardano-cli/test/Test/golden/shelley/node-pool/operator.vkey

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "StakePoolVerificationKey_ed25519",
+    "description": "Stake Pool Operator Verification Key",
+    "cborHex": "58204284e9702a97561fbb0c91789b61636066e195c134a50ba65b4036c20a97f383"
+}
```

Comment:
Probably worth adding a newline to these and making sure they always get created with a newline.

Reply:
Done!